products/ga_s2_clear_pixel_count.yml

```diff
@@ -1,4 +1,4 @@
-name: ga_s2_clear_pixel_count
+name: pc_s2_annual
 description: Clear Pixel Count S2A/B
```

Comment:
And this, please!

Reply:
Probably best to rename yaml file as well to match the new name.

Reply:
Great idea. Also, keep it to the new convention `<product_name>.odc-product.yaml`

Reply:
Done.

Reply:
Can we update the description to align with the geomedian product. Same structure :-)